src/anybase/anycml_func_std.py

```diff
@@ -429,6 +430,8 @@ def Import(_xParser, _lArgs, _lArgIsProc, *, sFuncName, funcGetCustomVarsFromPat
     # endif
 
     pathImport = path.ProvideReadFilepathExt(pathFile, [".json", ".json5", ".ison"], bDoRaise=True)
```

Comment:
I don't think that this function can deal with wildcards, and I am not sure what happens if you pass wildcards to it.

Reply:
Thats correct, it seems at least on my OSD machine, path.ProvideReadFilepathExt ignores wildcards altogether, which is why the glob function in line 434 is used to resolve the wildcards present in the output of path.ProvideReadFilepathExt

The only problematic case I see is a "*" character present in the actual filename of the file to be imported